app/views/application/index.html.slim

```diff
@@ -1,3 +1,4 @@
 .text-center
+  = fa_icon "heart"
```

Comment:
yes, this was to test that font awesome was working. do you want me to remove it? or should we leave it?